Makefile

```diff
@@ -8,3 +8,9 @@ lint:
 
 dev:
 	$(SRC)/manage.py runserver 127.0.0.1:8000
+
+builddeb:
+	dpkg-buildpackage -us -uc -b
```

Comment:
Consider using `debuild` which will also run `lintian`. Refer to [Chapter 6](https://www.debian.org/doc/manuals/maint-guide/build.en.html) from Debian New Maintainers' Guide.

You can use `gbp dch` from [`git-buildpackage`](https://honk.sigxcpu.org/piki/projects/git-buildpackage/) to increment the debian changelog based on git tags, should you choose to use them.